### PR TITLE
proof of concept improving user info experience

### PR DESCRIPTION
**Pull request by ikwattro**
Currently, user info display for actions results, like `Content updated` are displayed by a `alert` JS function.

As you already use bootstrap, my proposition is to replace these old-time alerts by the bootstrap ones.

See screencast below, if you're happy with it I will change it for all the alerts.

img

### CODE REVIEW

**Comment by jeremykendall** [comment]
Yes, go for it. I was going to do that but haven't gotten around to it. Thanks!

File: templates/layout.mustache [hunk 1]
```diff
@@ -43,6 +43,9 @@
             </div>
         </div>
         <div class="container">
+        <div class="col-sm-12" id="user-alert">
+            <div id="alert-block" class="alert" role="alert"></div>
+        </div>
             {{$ content }}{{/ content }}
```
Comment (jeremykendall) [inline templates/layout.mustache:49]: I love that this is in `layout.mustache` and therefore available everywhere. Nicely done.

Comment (ikwattro) [inline templates/layout.mustache:49]: yeah otherwise you would repeat this block on all tpls.

Christophe Willemsen

**Comment by ikwattro** [comment]
done ;-)

**Comment by jeremykendall** [comment]
:+1: 

I have some things to merge myself this afternoon. Once those are in, I'll pull this in. Thanks again.

**Comment by ikwattro** [comment]
Ok, so maybe I'll need to rebase my branch depending on your changes before you merge.

**Comment by jeremykendall** [comment]
That's a good point, and something I'd like to avoid, because conflicts are not fun. Merging now.

**Pull request merged**